Cannot find module 'tslint'

node version:  **v8.9.3**
npm version: **v.6.1.0**

**Reproduce steps:**
1. Clone the repository (master branch https://github.com/tech-conferences/confs.tech/commit/73dd5607ed3896088d6450c33b778451b8612d72)
2. Run `npm i`
3. Run `npm start`

**Output:**
```
/Users/mike/workspace/confs.tech/node_modules/fork-ts-checker-webpack-plugin/lib/types/NormalizedMessage.d.ts
ERROR in /Users/mike/workspace/confs.tech/node_modules/fork-ts-checker-webpack-plugin/lib/types/NormalizedMessage.d.ts(2,30):
TS2307: Cannot find module 'tslint'.
```

--- a/package.json
+++ b/package.json
@@ -90,6 +90,7 @@
     "@types/react-dom": "^16.0.6",
     "@types/react-helmet": "^5.0.6",
     "@types/react-router-dom": "^4.3.0",
+    "@types/webpack": "4.4.10",
     "babel-plugin-transform-es2015-modules-commonjs": "^6.26.0",
     "babel-preset-env": "^1.6.1",
     "eslint": "^4.5.0",
@@ -103,6 +104,7 @@
     "prettier": "^1.5.3",
     "sass-loader": "^6.0.6",
     "source-map-loader": "^0.2.3",
+    "tslint": "5.11.0",
     "typescript": "^2.9.2"
   },
   "homepage": "https://confs.tech",

